### DISCUSSION TITLE
Batch convert files that have the same name. All named "**** Seq''.

### DISCUSSION THREAD

**Discussion by berni4** [vgmtrans/vgmtrans#517] [Q&A]
Hello, 

On some sound rom files, mainly on ps1, ps2 and ds. all sample/seq etc files all have the same name, or vgmtrans cannot read there name so it gives all of them the same name. So when loading into vgmtrans i end up with 100 or so files all named ''sony PS2 Seq'' problem is that i cannot batch output these files on windows. I first need to manually rename all of them inside vgmtrans so they can live in the same folder on windows.

I think vgmtrans tries to batch output all these files but windows hard deletets whatever file that was overwritten. So when batch converting i always end up with only the last .midi and .sf2 of the selection i tried to convert.

I searched for a solution on windows to somehow recover the hard deleted files (if there is such a thing) or make windows autorename whatever file is overwriting a file with the same name but i had no succes.

Is there something else i could try?

Thanks allot.

(windows 10, latest vgmtrans 1.1)

**Comment by mikelow** [comment]
This is a very valid problem we need to address. I will work on a fix.
